src/public/modules/forms/admin/controllers/edit-fields-modal.client.controller.js

```diff
@@ -57,6 +57,17 @@ function EditFieldsModalController(
     vm.field.manualOptions = vm.field.fieldOptions
   }
 
+  // Serialize allowed email domains
+  if (vm.field.fieldType === 'email') {
+    const userEmailDomain = '@' + Auth.getUser().email.split('@').pop()
```

Comment:
`Auth.getUser()` has a chance to return `null`, so might want to do a check if user exists first. Maybe set it as a controller variable outside of this function? Maybe in line 53:

```ts
// Remember to inject $state into the dependencies of the controller
vm.user = Auth.getUser() || $state.go('signin')
...
// Then in this function
if (vm.field.fieldType === 'email') {
    const userEmailDomain = '@' + vm.user.email.split('@').pop()
...
```

Can search the codebase for other occurrence of using `Auth.getUser()`

Reply:
Noted! Thanks for the catch